build/tasks/build-task.coffee

```diff
@@ -145,9 +145,9 @@ module.exports = (grunt) ->
     cp 'src', path.join(appDir, 'src'), filter: /.+\.(cson|coffee)$/
     cp 'static', path.join(appDir, 'static')
 
-    cp path.join('apm', 'node_modules', 'atom-package-manager'), path.join(appDir, 'apm'), filter: filterNodeModule
+    cp path.join('apm', 'node_modules', 'atom-package-manager'), path.resolve(appDir, '..', 'apm'), filter: filterNodeModule
```

Comment:
Since the apm path is changing, does this need to be updated? https://github.com/atom/atom/blob/dd271aae960039b35ce20e2ecfcfc5bc751b42da/src/package-manager.coffee#L132